tests/unit/aws/test_connect.py

```diff
@@ -92,7 +92,7 @@ def test_external_client_credentials_origin(self, mock, monkeypatch):
         )
         mock.assert_called_once_with(
             service_name="def",
-            region_name="us-east-1",
+            region_name=region_name,
```

Comment:
I'm not so sure we want to do this with unit tests. The behavior of the underlying region lookup seems to be not as well defined here :thinking:

Reply:
You're right, I picked this from https://github.com/localstack/localstack/pull/8204 but I'm unable to reproduce the original reason for the change. Reverted for now, I'll revisit this if it reoccurs.